app/src/main/res/layout/story_chapter_view.xml

```diff
@@ -4,57 +4,121 @@
 
   <data>
 
+    <import type="android.view.View" />
+
+    <import type="org.oppia.app.model.ChapterPlayState" />
+
     <variable
       name="viewModel"
       type="org.oppia.app.story.storyitemviewmodel.StoryChapterSummaryViewModel" />
   </data>
 
-  <androidx.constraintlayout.widget.ConstraintLayout
-    android:layout_width="350dp"
-    android:layout_height="wrap_content"
-    android:layout_marginStart="10dp"
-    android:layout_marginTop="10dp"
-    android:padding="16dp"
-    android:onClick="@{viewModel.onExplorationClicked}"
-    android:contentDescription="@{viewModel.name}"
-    app:roundedRectDrawableWithColor="@{viewModel.chapterThumbnail.backgroundColorRgb}">
-
-    <ImageView
-      android:id="@+id/chapter_thumbnail"
+  <FrameLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <com.google.android.material.card.MaterialCardView
```

Comment:
If you check in detail the corners of the card you would find a white space on both top corners, and white card corner below the stroke corner borders of bottom rounded corners.

Reply:
I am not able to find a proper solution for this as of now, I have mentioned this in the issues checklist, so it would nice if we can merge this PR and fix that issue later on.